src/Tribe/Main.php

```diff
@@ -1756,12 +1757,51 @@ public function addVenueAndOrganizerEditor() {
 		}
 
 		/**
+		 * Display a helper for the user, about the location and microdata for rich snippets
+		 * @param int $postId the event ID to see if the helper is needed
```

Comment:
Also - should be `$post_id` (rather than `$postId`) to match the method signature